submissions/maxovsanyuk/js-dom/js/data.js

```diff
@@ -0,0 +1,111 @@
+let newItems = ['General Basics','Front-End Basics', 'Advanced Topics',];
+
+for (const value of newItems){
+    let topNav = document.createElement('div');
+    topNav.innerHTML = "<span>" + value + "</span>";
+    menu.appendChild(topNav);
+    topNav.className = "newItem"
+
+  
+    topNav.addEventListener("mouseover",function(){
+        topNav.classList.add('itemsMouseover');
+    });
+    topNav.addEventListener("mouseout",function(){
+        topNav.classList.remove('itemsMouseover');
+    });
+};
+
+let titleKottans = document.createElement("h1")
+document.getElementById("content").insertAdjacentHTML('beforeend','<h1>Kottans Front-End Course 2019</h1>');
+
+let content0 = document.getElementById("menu").children[0];
```

Comment:
First, naming as `content0`:

1) Don't use numbers in names of variables. In 99% of cases, you don't need them.
2) Name of variables must be declarative and when you read this name, you must understand what in this variable. So `content0` - is not declarative.
3) `document.getElementById("menu").children[0]` - bad line. First of all, you can use the specific method `Node#firstChild`, instead of `children[0]`. Second - sticking to the structure of your DOM tree is a bad idea because if you will add changes to DOM, your code will be broken. So pls replace this line with the selector, like `querySelector` or `getElementsByClassName`, or smth. else.

Reply:
```suggestion
let generalBasics = document.querySelectorAll("nav >div.new-item")[0];
```
Hi, can i solve it this way ?

Reply:
This is the same logic as it was before. Pls, see my new comments to your changes.